### PR TITLE
Add ExtraVanillaItems and ExtraVanillaBlocks to the CreativeInventory

### PR DESCRIPTION
**Pull request by kaxyum**

### CODE REVIEW

File: src/Main.php [hunk 1]
```diff
@@ -55,6 +55,8 @@ private static function registerSimpleBlock(string $id, Block $block, array $str
 		GlobalBlockStateHandlers::getDeserializer()->mapSimple($id, fn() => clone $block);
 		GlobalBlockStateHandlers::getSerializer()->mapSimple($block, $id);
 
+        	LegacyBlockIdToStringIdMap::getInstance()->add($id, $block->getTypeId());
```
Comment (dktapps) [inline src/Main.php]: This is not correct

File: src/Main.php [hunk 2]
```diff
@@ -48,9 +49,13 @@ public static function registerItems() : void{
 	private static function registerSimpleBlock(string $id, Block $block, array $stringToItemParserNames) : void{
 		RuntimeBlockStateRegistry::getInstance()->register($block);
 
+        	CreativeInventory::getInstance()->add($block->asItem());
+
 		GlobalBlockStateHandlers::getDeserializer()->mapSimple($id, fn() => clone $block);
 		GlobalBlockStateHandlers::getSerializer()->mapSimple($block, $id);
 
+		LegacyBlockIdToStringIdMap::getInstance()->add($id, $block->getTypeId());
```
Comment (JavierLeon9966) [inline src/Main.php]: Type ID is not the same as legacy ID

**Review by JavierLeon9966 — CHANGES_REQUESTED** [review]

File: src/Main.php (hunk 2 above)
Comment (kaxyum) [reply src/Main.php]: i'm idiot

File: src/Main.php [hunk 3]
```diff
@@ -54,7 +55,7 @@ private static function registerSimpleBlock(string $id, Block $block, array $str
 		GlobalBlockStateHandlers::getDeserializer()->mapSimple($id, fn() => clone $block);
 		GlobalBlockStateHandlers::getSerializer()->mapSimple($block, $id);
 
-		LegacyBlockIdToStringIdMap::getInstance()->add($id, $block->getTypeId());
+		LegacyBlockIdToStringIdMap::getInstance()->add($id, BlockTypeIds::newId());
```
Comment (dktapps) [inline src/Main.php]: no

**Pull request closed without merging**